.github/workflows/build.yml

```diff
@@ -14,22 +14,18 @@ on:
 
 jobs:
     tests:
-        runs-on: ubuntu-20.04
+        runs-on: ubuntu-latest
 
-        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
+        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}, Node ${{ matrix.node }}"
 
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.1"]
-                symfony: ["^5.4", "^6.0"]
-                sylius: ["~1.11.0", "~1.12.0", "1.13.x-dev as 1.13.0"]
-                node: ["14.x"]
-                mysql: ["8.0.36"]
-
-                exclude:
-                    - sylius: ~1.11.0
-                      symfony: "^6.0"
+                php: [ "8.1", "8.2", "8.3" ]
+                symfony: [ "^5.4", "^6.0" ]
+                sylius: [ "^1.12", "^1.13" ]
```

Comment:
`^1.12` installs the Sylius 1.13, so now version 1.12 is not tested at all, you need to change that to `~1.12.0`